android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -752,4 +840,310 @@ public static ErcToken createEthereumErcToken() {
         }
         return tokens;
     }
+
+    public static String getAccountName(AccountInfo[] accounts, String address) {
+        String addressLowerCase = address.toLowerCase(Locale.getDefault());
+        for (AccountInfo account : accounts) {
+            if (account.address.toLowerCase(Locale.getDefault()).equals(addressLowerCase)) {
+                return account.name;
+            }
+        }
+
+        return stripAccountAddress(address);
+    }
+
+    public static void openTransaction(
+            TransactionInfo txInfo, EthJsonRpcController ethJsonRpcController, Activity activity) {
+        if (txInfo == null || txInfo.txHash == null || txInfo.txHash.isEmpty()) {
+            return;
+        }
+        assert ethJsonRpcController != null;
+        ethJsonRpcController.getChainId(chainId -> {
+            ethJsonRpcController.getAllNetworks(networks -> {
+                for (EthereumChain network : networks) {
+                    if (!chainId.equals(network.chainId)) {
```

Comment:
this seems strange, but the desktop has the same `EthJsonRpcController::GetBlockTrackerUrlFromNetwork`